src/playground/Insights/EventsList/index.tsx

```diff
@@ -0,0 +1,36 @@
+import { Event, EventProps } from "../ui/event"
+
+export interface EventsListProps {
+  events: EventProps[]
+  title: string
```

Comment:
Title should be optional?

Reply:
True, good find, I'll change it now. Keep in mind I will change the composables as I work with the following insights which require their own specifics. These are not the final versions, it's all a WiP.